pkg/k8sutil/kotsadm.go

```diff
@@ -160,6 +161,11 @@ func UpdateKotsadmIDConfigMap(kotsadmID string) error {
 }
 
 func FindKotsadm(clientset *kubernetes.Clientset, namespace string) (string, error) {
+	if persistence.IsSQlite() {
```

Comment:
TODO: need to have a separate function for detecting the kots run workflow so that this isn't confusing (applies to other occurrences as well)